In [1]:
from scipy.io import netcdf
import numpy as np

In [2]:
nc = netcdf.netcdf_file("data/era-int_1979-2019-mm-l70-u-reg.nc")

The coordinates of Singapur are 1N, 104E

In [3]:
longitudes = np.array(nc.variables["longitude"][:])
np.where((longitudes < 104.5) & (longitudes > 103.5)) # +- tolerance

(array([148], dtype=int64),)

In [4]:
latitudes = np.array(nc.variables["latitude"][:])
np.where((latitudes < 1.5) & (latitudes > 0.5))

(array([126], dtype=int64),)

So the values for Singapur are at the indices longitude = 148 and latitude = 126